### PR TITLE
make transactions table safe;

### PR DESCRIPTION
**Pull request by anjor**
**Goals (and why)**:
transactions2 table is marked safe, but transactions table isn't.

**Implementation Description (bullets)**:
Marks transactions table safe.

**Testing (What was existing testing like?  What have you done to improve it?)**:

**Concerns (what feedback would you like?)**:

**Where should we start reviewing?**:

**Priority (whenever / two weeks / yesterday)**:

### CODE REVIEW

File: atlasdb-client/src/main/java/com/palantir/atlasdb/transaction/impl/TransactionConstants.java [hunk 1]
```diff
@@ -58,6 +59,7 @@ public static long getTimestampForValue(byte[] encodedTimestamp) {
     public static final TableMetadata TRANSACTION_TABLE_METADATA = TableMetadata.internal()
             .singleRowComponent("write_ts", ValueType.VAR_LONG)
             .singleNamedColumn(COMMIT_TS_COLUMN_STRING, "commit_ts", ValueType.VAR_LONG)
+            .nameLogSafety(LogSafety.SAFE)
             .build();
```
Comment (jeremyk-91) [inline atlasdb-client/src/main/java/com/palantir/atlasdb/transaction/impl/TransactionConstants.java:63]: uh can we add SweepStrategy.NOTHING here for consistency?

File: changelog/@unreleased/pr-5526.v2.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+type: improvement
+improvement:
+  description: |-
+    Make transactions table safe to log.
+    <!---
+    Please remember to:
+    - Add any necessary release notes (including breaking changes)
+    - Make sure the documentation is up to date for your change
+    --->
```
Comment (jeremyk-91) [inline changelog/@unreleased/pr-5526.v2.yml:9]: nit: remove autogenerated thing

**Review by jeremyk-91 — COMMENTED** [review]
looks good! minor bits

File: atlasdb-client/src/main/java/com/palantir/atlasdb/transaction/impl/TransactionConstants.java (hunk 1 above)
Comment (anjor) [reply atlasdb-client/src/main/java/com/palantir/atlasdb/transaction/impl/TransactionConstants.java:64]: done

File: changelog/@unreleased/pr-5526.v2.yml (hunk 2 above)
Comment (anjor) [reply changelog/@unreleased/pr-5526.v2.yml]: done

**Review by jeremyk-91 — APPROVED** [review]

**Pull request merged**